Commands/marvin_product/LintCommands.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Drush\Commands\marvin_product;
 
-use Drush\Commands\marvin\Lint\CommandsBase as LintCommandsBase;
+use Drush\Commands\marvin\LintCommandsBase as LintCommandsBase;
```

Comment:
The alias and name of the original class is the same. So here we don't need the alias. (... as LintCommandsBase)